schemas/content/product.schema.json

```diff
@@ -17,9 +18,8 @@
         "@id": {
           "title": "Identifier",
           "type": "string",
-          "format": "uri",
```

Comment:
@id if used must be a URI

Reply:
Also, it's much easier to turn any string into a URI than to provide the right context for a string ID to make it resolvable.

Reply:
As discussed with @trieloff changed back to `@id` & `uri`, same for category identifier.